Makefile

```diff
@@ -26,13 +26,21 @@ help: Makefile
 
 ## test-data
 testDataDir := eppo/src/androidTest/assets/
+tempDir := ${testDataDir}/temp
+gitDataDir := ${tempDir}/sdk-test-data
+branchName := main
+githubRepoLink := https://github.com/Eppo-exp/sdk-test-data.git
 .PHONY: test-data
 test-data: 
 	rm -rf $(testDataDir)
```

Comment:
`tempDir` is never created. I am going with in js repo:

```
	rm -rf $(testDataDir)
	rm -rf $(tempDir)
	mkdir -p $(testDataDir)
	mkdir -p $(tempDir)
```

Reply:
When we create `tempDir ` is also creates `testDataDir ` because it is the sub-folder.